refactor: :recycle: update roadmap based on latest changes (#347)
seedcase-project/seedcase-project: roadmap/index.qmd

| {{< var status.wip >}} | [`guidebook.seedcase-project.org`](https://guidebook.seedcase-project.org/): Guidebook to doing research software engineering in a team-setting. |
| {{< var status.wip >}} | [`sprout.seedcase-project.org`](https://sprout.seedcase-project.org/docs/guide) usage guide: Using and interacting with `seedcase-sprout`. |
| {{< var status.planned >}} | Flower usage guide: Using and interacting with `seedcase-flower`. |
| {{< var status.planned >}} | Propagate usage guide: Using and interacting with `seedcase-propagate`. |
| {{< var status.planned >}} | Garden usage guide: Using and interacting with `seedcase-garden`. |
| {{< var status.planned >}} | Shears usage guide: Using and interacting with `seedcase-shears`. |

: "Wishlist" of goal-type documentation products.

::: content-hidden
Potentials:

-   Improving the research software development lifecycle by integrating
    DevOps practices.
-   A guided walkthrough: Team-based collaborative practices and
    workflows in a research environment.
-   Using DocOps workflows and infrastructure to reduce time between
    writing and dissemination.

> -   Building DataOps pipelines and infrastructure for improving research

Reviewer: ```suggestion
-   Building DataOps pipelines and infrastructure to improve research
```
(or change "reduce" below)

| Status | Deliverable |
|:--------------------------------------------:|:-------------------------|

> | {{< var status.ongoing >}} | [`community`](https://github.com/seedcase-project/community): Content for community building, outreach, and news for the Seedcase Project. |

Reviewer: ```suggestion
| {{< var status.ongoing >}} | [`community`](https://github.com/seedcase-project/community): Content for community building, outreach, and news about the Seedcase Project. |
```
Maybe?

> | {{< var status.ongoing >}} | [`decisions`](https://github.com/seedcase-project/decisions): Archival records on the decisions made for tools or processes we use in the Seedcase Project. |

Reviewer: ```suggestion
| {{< var status.ongoing >}} | [`decisions`](https://github.com/seedcase-project/decisions): Archival records of decisions made about tools or processes we use in the Seedcase Project. |
```
